### ISSUE TITLE
Vocabulary에 최신 용어집 (KCD8 등) 반영 여부

### ISSUE THREAD

**Issue by jaeyounglee978** [ohdsi-korea/OmopVocabularyKorea#13]
안녕하세요. 혹시 OHDSI 및 OHDSI-Korea에서 vocabulary 데이터베이스에 kcd8을 추가할 계획이 혹시 있으신지 알 수 있을까요? 현재 athena 에서는 kcd7까지만 확인되고 별도 작업 중인 내역을 확인하기는 어려운 거 같아서 이슈로 질문 남깁니다.

**Comment by chandryou** [comment]
예 문의 감사합니다. 필요하다면 KCD8 업로드 진행할 의향이 있습니다. ^^

**Comment by jaeyounglee978** [comment]
@chandryou 확인 감사합니다 교수님! KCD8이 등록된다면, 현재 ohdsi vocabulary-5 repo에 등록된 kcd7처럼 타 용어집과의 relation까지 등록되는지, 그리고 [athena](https://athena.ohdsi.org/)에도 반영되는 것인지도 알 수 있을까요?
 추가적으로, ohdsi 에서 관리하는 vocabulary는 여러 용어집을 포함하고 있는데, 이것이 각 용어집의 subset만 포함하는지 혹은 전체 용어를 전부 import하는지도 질문드리고 싶습니다.

감사합니다.

**Comment by chandryou** [comment]
>ohdsi vocabulary-5 repo에 등록된 kcd7처럼 타 용어집과의 relation까지 등록되는지, 그리고 [athena](https://athena.ohdsi.org/)에도 반영되는 것인지도 알 수 있을까요?

예 가능합니다. 현재 KCD7 매핑이 불완전한 상태인데- KCD8 추가적인 mapping을 공유해주시면 충분히 반영 가능합니다.

>추가적으로, ohdsi 에서 관리하는 vocabulary는 여러 용어집을 포함하고 있는데, 이것이 각 용어집의 subset만 포함하는지 혹은 전체 용어를 전부 import하는지도 질문드리고 싶습니다.

질문에 대한 예시가 있으면 답변드리기 더 좋겠는데요, 일단 케바케라고 생각하시면 되지 않을까 합니다. ^^
